### PR TITLE
Move to using external CSV sources for database

### PR DESCRIPTION
**Pull request by mpuckett159**
This update moves to importing the data for the database from an external source so we don't have to track the CSV files in git anymore. The data sources are a DigitalOcean Space owned by techbloc. The file names are the same but prepended with `https://techblocsea.sfo3.digitaloceanspaces.com/spd-lookup/`.

### CODE REVIEW

File: db/Dockerfile [hunk 1]
```diff
@@ -1,9 +1,9 @@
 FROM postgres:alpine
 
+RUN apk --update add curl
```
Comment (AetherUnbound) [inline db/Dockerfile:3]: Are there any caches that we need to clear after this to reduce image size?

File: db/sql/10_thurstoncty.sql [hunk 2]
```diff
@@ -8,4 +8,4 @@ CREATE TABLE IF NOT EXISTS thurston_officers (
 );
 
 COPY thurston_officers (title,last_name,first_name,call_sign,call_sign_2)
-FROM '/seed/ThurstonCounty-WA-Sheriffs-Office_05-01-21.csv' DELIMITER ',' CSV HEADER;
+FROM PROGRAM 'curl "https://techblocsea.sfo3.digitaloceanspaces.com/spd-lookup/ThurstonCounty-WA-Sheriffs-Office_05-01-21.csv"' DELIMITER ',' CSV HEADER;
```
Comment (AetherUnbound) [inline db/sql/10_thurstoncty.sql:11]: I wonder if these URLs could be templated...maybe then we could have these in minio locally for testing. What do you think?

**Review by AetherUnbound — COMMENTED** [review]
Sorry it's taking me so damn long to get to this! Do you mind also adding documentation for how you would suggest updating the CSVs?

File: db/sql/10_thurstoncty.sql (hunk 2 above)
Comment (mpuckett159) [reply db/sql/10_thurstoncty.sql:11]: Oh that's a good idea, I bet we can. I'll look into it.

File: db/Dockerfile (hunk 1 above)
Comment (mpuckett159) [reply db/Dockerfile:3]: Yup `--no-cache` saves us 2MB, nice.

File: db/sql/10_thurstoncty.sql (hunk 2 above)
Comment (mpuckett159) [reply db/sql/10_thurstoncty.sql]: Took quite a bit longer than I meant it to, but I've gotten this working. Will expand more on thoughts in another comment.

**Comment by mpuckett159** [comment]
So I think what we should do is have a folder in `https://techblocsea.sfo3.digitaloceanspaces.com/spd-lookup/` just called something like latest or whatever, and we can just have files named simply in there like "seattle.csv" or something, and then we can store the historical rosters in another folder in case we need them, that way when a new roster is uploaded all we need to do is restart the container to consume it.

**Comment by AetherUnbound** [comment]
> So I think what we should do is have a folder in `https://techblocsea.sfo3.digitaloceanspaces.com/spd-lookup/` just called something like latest or whatever, and we can just have files named simply in there like "seattle.csv" or something, and then we can store the historical rosters in another folder in case we need them, that way when a new roster is uploaded all we need to do is restart the container to consume it.

I love this idea! Should be able to set that up 👍🏼

File: .env.example [hunk 3]
```diff
@@ -6,3 +6,5 @@ DB_PASSWORD=postgres
 DB_IMAGE_TAG=latest
 # Default will likely be "latest" but can also be any tag available for the image hosted at https://ghcr.io/orcacollective/spd-lookup-api
 API_IMAGE_TAG=latest
+# Roster source can be used to point at a local source for rosters for testing purposes, production should always point to the default value below
+ROSTER_SOURCE=https://techblocsea.sfo3.digitaloceanspaces.com/spd-lookup
```
Comment (AetherUnbound) [inline .env.example]: We should make this default NOT our actual production 😅  Probably just a shim value.

**Comment by mpuckett159** [comment]
Alright this should be g2g now.

File: docker-compose.yml [hunk 4]
```diff
@@ -10,6 +10,7 @@ services:
       - POSTGRES_USER=${DB_USERNAME:-postgres}
       - POSTGRES_PASSWORD=${DB_PASSWORD}
       - POSTGRES_DB=ACAB_DB
+      - ROSTER_SOURCE=${ROSTER_SOURCE:-https://techblocsea.sfo3.digitaloceanspaces.com/spd-lookup/latest}
```
Comment (AetherUnbound) [inline docker-compose.yml:13]: Same for the reference here. And we probably don't need a default here as well if it's in the .env.

File: .env.example (hunk 3 above)
Comment (mpuckett159) [reply .env.example]: The prod source here is just a read only source, so I figured pointing people there by default was the best bet for "pull this down and spin it up if you want to" scenarios.

Comment (AetherUnbound) [reply .env.example]: That's fair...and I guess we have techbloc references elsewhere 🤷🏼‍♀️

**Review by AetherUnbound — APPROVED** [review]

File: .env.example (hunk 3 above)
Comment (AetherUnbound) [reply .env.example]: Oh but wait, should this have `/latest` on the end like the other reference?

Comment (mpuckett159) [reply .env.example]: ye it's updated I think we're just commenting on an out of dated bit of code.

Comment (AetherUnbound) [reply .env.example]: 🤦🏼‍♀️ that's what I get for multitasking, sorry!

**Pull request merged**